src/main/kotlin/com/sourcegraph/cody/agent/protocol/ContextItem.kt

```diff
@@ -83,11 +83,11 @@ data class ContextItemFile(
         if (isLocal()) {
           "@${uri.path.removePrefix(projectPath ?: "")}"
         } else {
-          val repoCommitFile = uri.path.split("@", "/-/blob/")
-          if (repoCommitFile.size == 3) {
-            val repo = repoCommitFile[0].split("/").lastOrNull()
-            "$repo ${repoCommitFile[2]}"
-          } else uri.path
+          val repo = repoName?.split("/")?.lastOrNull()
+          val file = title
```

Comment:
Nit: 
You reassign to `file`, but do a check for `title`.
It is still guaranteed to have a correct result as both are `val`s, but looks a bit weird.